### PR TITLE
Alerting: Show receiver in groups view to avoid duplication in the list

### PR DESCRIPTION
**Pull request by soniaAguilarPeiron**
**What is this feature?**

In the Groups page, sometimes groups appear to be duplicated. This may lead to confusion. 
The reason behind this issue is that alert groups are grouped based on the receivers they are associated with. 
However, this receiver information is not displayed in the list, which can result in a misunderstanding of these groups.

This pull request addresses this issue by including the receiver name for each group when no additional grouping has been selected. 

**Why do we need this feature?**

Alert group page show display an accurate list.

**Who is this feature for?**

All users.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/support-escalations/issues/8022

**Special notes for your reviewer:**

**When no additional grouping selected** 

Screenshot 2023-10-25 at 16 38 30

**When additional grouping is selected**

Screenshot 2023-10-25 at 16 05 36

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by gillesdemey** [comment]
Can we use the same styles as the notification policies for the "delivered to"?
image

**Comment by soniaAguilarPeiron** [comment]
> Can we use the same styles as the notification policies for the "delivered to"? image


Done!

File: public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx [hunk 1]
```diff
@@ -31,7 +33,15 @@ export const AlertGroup = ({ alertManagerSourceName, group }: Props) => {
             data-testid="alert-group-collapse-toggle"
           />
           {Object.keys(group.labels).length ? (
-            <AlertLabels labels={group.labels} size="sm" />
+            <Stack direction="row">
```
[the remark this answers is not in the document]
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx:36]: Oh! nice! I didn't know that one! going to update it

File: public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx [hunk 2]
```diff
@@ -65,8 +76,10 @@ const getStyles = (theme: GrafanaTheme2) => ({
     align-items: center;
   `,
   summary: css``,
-  spanElement: css`
+  receiver: css`
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx:79]: Don't think we need this bit of CSS :)

Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx:79]: true!

Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx:79]: true!

**Review by gillesdemey — APPROVED** [review]
LGTM

**Pull request merged**